src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6247,4 +6247,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_BadInheritanceFromRecord" xml:space="preserve">
     <value>Only records may inherit from records.</value>
   </data>
+  <data name="ERR_BadRecordMember" xml:space="preserve">
```

Comment:
Consider choosing a more specific name if the diagnostic only applies to properties for positional parameters, because we will may need similar diagnostics for other synthesized members.

Reply:
Done